unit-files/ping-pong-sidekick@.service

```diff
@@ -1,15 +1,15 @@
 [Unit]
 Description=Register for Ping-Pong
-BindsTo=ping-pong.service
-After=ping-pong.service
+BindsTo=ping-pong@%i.service
+After=ping-pong@%i.service
 
 [Service]
 EnvironmentFile=/etc/environment
 RemainAfterExit=yes
-ExecStart=/bin/sh -c "/bin/etcdctl set \"/vulcand/upstreams/ping-pong/endpoints/ping-pong-sidekick.service\" http://$COREOS_PUBLIC_IPV4:8880; \
+ExecStart=/bin/sh -c "/bin/etcdctl set \"/vulcand/upstreams/ping-pong/endpoints/ping-pong-sidekick@%i.service\" http://$COREOS_PUBLIC_IPV4:8880; \
```

Comment:
You don't need to hardcode the `ping-pong-sidekick` part here. Systemd gives you even more variables, like `%n` for the unitname.